apps/enmeshed/lib/splash_screen.dart

```diff
@@ -90,9 +90,12 @@ class _SplashScreenState extends State<SplashScreen> {
         databaseFolder: './database',
       ),
     );
-    GetIt.I.registerSingletonAsync<EnmeshedRuntime>(() async => runtime.run());
 
-    await GetIt.I.allReady();
+    final result = await runtime.run();
+
+    if (result.isError) return router.go('/general-error');
+
+    GetIt.I.registerSingleton(result.value);
```

Comment:
just register the runtime
```suggestion
    GetIt.I.registerSingleton(runtime);
```